h2o/h2o.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+
+echo "BEGIN Stage 1 : Install H2O libraries and dependencies"
+apt-get update
+
+
+
+## Set Spark and Sparkling water versions
+H2O_SPARKLING_WATER_VERSION="3.28.0.3-1"
```

Comment:
Consider using VM metadata and set default value if not provided.

Reply:
Good point.
Updated code accordingly.